src/unchunker.ts

```diff
@@ -98,6 +184,7 @@ class ChunkCollector {
      * Return whether this chunk collector already contains a chunk with the specified serial.
      */
     public hasSerial(serial: number): boolean {
+        // TODO: Performance. Slow?
```

Comment:
Should this be benchmarked?

Reply:
I guess we should compare it to a hash map approach (or even combined - array and hash map).